test/literal.ts

```diff
@@ -26,7 +24,6 @@ test("it should throw if a different type is given", t => {
 
     const error = t.throws(() => {
         Factory.create({ shouldBeOne: 2 })
-    })
-
-    t.is(error.message, `[mobx-state-tree] Value '{"shouldBeOne":2}' is not assignable to type: TestFactory, expected an instance of TestFactory or a snapshot like \'{ shouldBeOne: 1 }\' instead.`)
+    }, `[mobx-state-tree] Error while converting {"shouldBeOne":2} to TestFactory:
+at path "/shouldBeOne" snapshot 2 is not assignable to type: 1, expected an instance of 1 or a snapshot like '1' instead.`)
```

Comment:
same here : `snapshot '2'`? `type: 1`, is although a little weird, correct I think